cmd/zed/zed.c

```diff
@@ -1,6 +1,6 @@
 /*
  * This file is part of the ZFS Event Daemon (ZED)
```

Comment:
nit: You lost the period at the end of this sentence (and in the other files too).

Reply:
Fixed